src/core/drive/morph_renderer.js

```diff
@@ -57,9 +57,9 @@ export class MorphRenderer extends PageRenderer {
         })
 
         return !event.defaultPrevented
-      } else {
-        return false
       }
+      
+      return false
```

Comment:
The `else` was redundant here 🤧

Reply:
Hey @omarluq, you are right it's redundant but I'd rather keep the PR focused on its purpose without performing cosmetic changes around 🙏.